src/TwMgr.cpp

```diff
@@ -1904,16 +1904,17 @@ int ANT_CALL TwInit(ETwGraphAPI _GraphAPI, void *_Device)
     _CrtSetDbgFlag(_CRTDBG_LEAK_CHECK_DF|_CrtSetDbgFlag(_CRTDBG_LEAK_CHECK_DF));
 #endif
 
-    if( g_TwMasterMgr!=NULL )
+    if( g_TwMasterMgr!=NULL || g_WndsPtr!=NULL )
```

Comment:
@gittyupagain explained to me that this was the source of the crash--using the `std::map` after it was destroyed--and its corresponding fix.